src/mathml.cpp

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright libCellML Contributors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include "mathml.h"
+
+namespace libcellml {
+
+bool compareMath(const std::string &math1, const std::string &math2)
+{
```

Comment:
Are we really going to say that differences in whitespace in a mathml string are enough to render it non-equal?  That seems strange to me, especially as we "prettify" it internally?

Reply:
945d4ec